src/queueserver.ts

```diff
@@ -291,13 +291,27 @@ export const submitPlan = async(submitPlan: ISubmitPlanObject): Promise<IPlanObj
             } else {
                 plan.kwargs[key] = Number(value[0])
             }
-        }
-      } else {
-        plan.kwargs[key] = value;
+	} else {
+		if (key == "md"){
+		plan.kwargs[key] = {"foo": "bar"};
+		} else {
+			plan.kwargs[key] = value;
+		}
+	}
       }
     }
-    
+
+
+    if (submitPlan.name === "xafs"){
+      plan.kwargs['bounds'] = "-30 40"
+      plan.kwargs['steps'] = "0.5"
+      plan.kwargs['times'] = "0.5"
+      plan.kwargs['snapshots'] = false
+      plan.kwargs['htmlpage'] = false
+      plan.kwargs['lims'] = false
+    }
```

Comment:
How can we make sure it's parametrized well enough so that we don't hard-code the BMM-specific things here?

Reply:
I think this can probably be made part of the XAFSPlanForm, which is the BMM specialization.
Also bounds, steps, and times can be removed by fixing the code that formats the lists to be sent to the server.